PackHistorian/View/PackNameConverter.cs

```diff
@@ -732,6 +752,46 @@ class PackNameConverter : IValueConverter {
           { Locale.thTH, "Madness at the Darkmoon Faire สีทอง" },
         }
       },
+      {
+        686, new Dictionary<Locale, string>() {
+          { Locale.enUS, "Golden Forged in the Barrens" },
+          { Locale.enGB, "Golden Forged in the Barrens" },
+          { Locale.frFR, "Forgés dans les Tarides doré" },
+          { Locale.deDE, "Geschmiedet im Brachland (Golden)" },
+          { Locale.koKR, "황금 불모의 땅" },
+          { Locale.esES, "Doradas de Forjados en los Baldíos" },
+          { Locale.esMX, "Dorado de Forjados en los Baldíos" },
+          { Locale.ruRU, "Золотой комплект «Закаленных Степями»" },
+          { Locale.zhTW, "贫瘠之地的锤炼金卡" },
+          { Locale.zhCN, "貧瘠之地金卡" },
+          { Locale.itIT, "Forgiati nelle Savane Dorata" },
+          { Locale.ptBR, "Forjado nos Sertões Dourado" },
+          { Locale.plPL, "Złoty zestaw Zahartowani przez Pustkowia" },
+          { Locale.ptPT, "Forjado nos Sertões Dourado" },
+          { Locale.jaJP, "ゴールデン荒ぶる大地の強者たち" },
+          { Locale.thTH, "Forged in the Barrens สีทอง" },
+	 }
+       },
+       {
+        688, new Dictionary<Locale, string>() {
+          { Locale.enUS, "Year of the Phoenix" },
+          { Locale.enGB, "Year of the Phoenix" },
+          { Locale.frFR, "L'Année Du Phénix" },
+          { Locale.deDE, "Jahr des Phönix" },
+          { Locale.koKR, "피닉스의 해" },
+          { Locale.esES, "Año de Phoenix" },
+          { Locale.esMX, "Año de Phoenix" },
+          { Locale.ruRU, "Год Феникса" },
+          { Locale.zhTW, "鳳凰年" },
+          { Locale.zhCN, "凤凰年" },
+          { Locale.itIT, "L'Anno Della Fenice" },
+          { Locale.ptBR, "Ano Da Fênix" },
+          { Locale.plPL, "Roku " },
```

Comment:
`Roku` for `year` but nothing for `Phoenix`?
The packs are unnamed but the card back is called `Rok Feniksa`.
(and various other differences, see [my extracted version](https://github.com/sgkoishi/PackTracker/blob/patch-1/PackHistorian/View/PackNameConverter.cs#L555-L594))

Reply:
Yes, I even left a space to add a word later, my mistake. In the near future I will fix some points and update the file